modules/tinymce/src/plugins/anchor/main/ts/core/FilterContent.ts

```diff
@@ -5,27 +5,28 @@
  * For commercial licenses see https://www.tiny.cloud/
  */
 
-const isAnchorNode = function (node) {
-  return !node.attr('href') && (node.attr('id') || node.attr('name')) && !node.firstChild;
+import { Arr } from '@ephox/katamari';
+import Node from 'tinymce/core/api/html/Node';
+import Editor from 'tinymce/core/api/Editor';
+
+const isAnchorNode = (node: Node) => {
```

Comment:
Another place that maybe should be called `isNamedAnchorNode`?

- [ ] Consider renaming this